src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -109,7 +109,6 @@ export interface ArtworkFilterContextProps {
   setCounts?: (counts: Counts) => void
 
   // Handlers
-  onArtworkBrickClick?: (artwork: any, props: any) => void
```

Comment:
Should we be deleting this API feature? It seems handy and for the time being can remain unused

Reply:
Looks like we've only ever used it for tracking, so I'm hoping that by removing there is less ambiguity on how to hook this up correctly.

Reply:
Cool, sounds good 👍